README.md

```diff
@@ -121,6 +121,7 @@ customSources: # Main usecase is probably to set tv media player to play TV soun
       thumbnail: https://cdn-icons-png.flaticon.com/512/716/716429.png
 skipAdditionalPlayerSwitches: true # default is false, which means additional switches will be shown in player if available (such as crossfade button)
 disableDynamicVolumeSlider: true # default is false. See more in section further down.
+prefixArtwork: http://192.168.0.59:8123 #default is ''. This is your HA local IP-adress and is used to make the artwork work if you cast the dashboard with Google Cast.
```

Comment:
Could we change the name of the property to `artworkHostname`?
Also, could we change the comment to?
```
#default is ''. Usually not needed, but depending on your setup your device might not be able to access the artwork on the default host. One example where it could be needed is if you cast the dashboard with Google Cast.
```

Reply:
Thank you for the feedback. I have changed the name of the property and the comment in the Readme.